app/src/main/res/values/styles.xml

```diff
@@ -70,4 +70,29 @@
         <item name="android:divider">@null</item>
     </style>
 
+    <style name="TextAppearance"/>
```

Comment:
FINALLLLLYYY it's as easy as it should have been all along

Reply:
It's sooo beautiful......